db/migrate/20200615150624_add_other_passenger_information_taken_to_incident_reports.rb

```diff
@@ -1,5 +1,5 @@
 class AddOtherPassengerInformationTakenToIncidentReports < ActiveRecord::Migration[5.2]
   def change
-    add_column :incident_reports, :other_passenger_information_taken, :boolean
+    add_column :incident_reports, :other_passenger_information_taken, :boolean, default: false
```

Comment:
Editing an existing migration will not change the database.

Reply:
I made this migration after removing the column. The schema reflects that the default has been made.